_episodes/02-generalsyntax.md

```diff
@@ -0,0 +1,264 @@
+---
+title: "General Syntax of a Shell Command"
+teaching: ??
+exercises: ??
+questions:
+- "How do I tell the shell to do something?"
+- "How do flags and options modify a shell program?"
```

Comment:
Perhaps this question could be rephrased as "How does using flags and options change the behaviour of a shell program?"
Flags and options do not change a shell program in terms of its content. By specifying "behaviour", we could reduce the confusion a learner may have over what flags and options do.